### DISCUSSION TITLE
Compiling mame for only X68000?

### DISCUSSION THREAD

**Discussion by HazukiXX** [mamedev/discussions#127] [Q&A] [answered]
I don't want to compile the entire mame because it's too large and takes too long. In the official "compilingmame" page, there are those instructions:

> Rebuilding MAME on a dual-core (e.g. i3 or laptop i5) machine:
> 
> `make -j3`
> 
> Rebuilding MAME for just the Pac-Man and Galaxian families of systems, with tools, on a quad-core (e.g. i5 or i7) machine:
> 
> `make SUBTARGET=pacem SOURCES=src/mame/pacman,src/mame/galaxian TOOLS=1 REGENIE=1 -j5`
> 
> Rebuilding MAME for just the Apple II systems, compiling up to six sources in parallel:
> 
>` make SUBTARGET=appulator SOURCES=apple/apple2.cpp,apple/apple2e.cpp,apple/apple2gs.cpp REGENIE=1 -j6`

What would be the equivalent for X68000? Note that I'd also want to make it compatible with the mt32 and sc55 sound sources through its -midiout.

**Comment by sairuk** [comment, marked as answer]
`SUBTARGET` is an arbitrary name in this case, all the machine support should come down as is with the machine and you should be able to identify what source files your need at a minimum with `mame x68000 -listsource` i believe

**Reply by HazukiXX** [reply]
Thanks for the info. The "-listsource" command was helpful.
